addons/sales_team/static/src/js/sales_team_dashboard.js

```diff
@@ -17,7 +17,7 @@ var _lt = core._lt;
 var SalesTeamDashboardView = KanbanView.extend({
     display_name: _lt('Dashboard'),
     icon: 'fa-dashboard',
-    searchview_hidden: true,
+    searchview_hidden: false,
```

Comment:
@tivisse 
Yes. that's main purpose of this task. to set "My Favorites" filter in the dashboard in search view.